app/code/community/Bolt/Boltpay/Controller/Traits/WebHookTrait.php

```diff
@@ -120,6 +120,7 @@ protected function sendResponse($httpCode, $data = array(), $exitImmediately = t
 
         if ($exitImmediately) {
             echo $content;
+            Mage::dispatchEvent('controller_front_send_response_after');
             exit;
```

Comment:
@mccottry there is method, best practice in Magento:
return $this->getResponse()->setBody(Mage::helper('core')->jsonEncode($arrayResponse));
Using "exit" this is .... bro.... you made me cry again

Reply:
The is very sound reason why the Magento mechanism is not being used and exit is utilized.
https://github.com/BoltApp/bolt-magento1/pull/524/files#diff-7f54a3a8b470138e5bdeed099526433dL112-L114
https://github.com/BoltApp/bolt-magento1/pull/524

More importantly, both topics have nothing to do with this PR.

Reply:
The is very sound reason why the Magento mechanism is not being used and exit is utilized.
https://github.com/BoltApp/bolt-magento1/pull/524/files#diff-7f54a3a8b470138e5bdeed099526433dL112-L114

https://github.com/BoltApp/bolt-magento1/pull/524

More importantly, both topics have nothing to do with this PR.